core/trino-main/src/main/java/io/trino/execution/scheduler/BinPackingNodeAllocatorService.java

```diff
@@ -128,7 +128,7 @@ public void start()
         }
 
         refreshNodePoolMemoryInfos();
-        executor.scheduleWithFixedDelay(this::refreshNodePoolMemoryInfos, 1, 5, TimeUnit.SECONDS);
+        executor.scheduleWithFixedDelay(this::refreshNodePoolMemoryInfos, 1, 1, TimeUnit.SECONDS);
```

Comment:
Any reason why still keeping two threads? Why not get the node pool memory infos in `processPendingAcquires`?

Reply:
Is it because you expect we will process pending acquires with less than 1s interval?